### PR TITLE
Add functional test for Page.View()

### LINKED ISSUE #5802: In a Razor page returning View from OnGet results in a compile error.
Having:

```
@page

@functions {
    public IActionResult OnGet()
    {
        Message = "From OnGet";

        return View();
    }
    public string Message { get; set; } = "Default";
}
<p>The message: @Message</p>
```

Results in a compile error:
image

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Fixes #5802.

### CODE REVIEW

File: test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs [hunk 1]
```diff
@@ -112,6 +112,22 @@ public RazorPagesTest(MvcTestFixture<RazorPagesWebSite.Startup> fixture)
         }
 
         [Fact]
+        public async Task ViewReturnsPage()
+        {
+            // Arrange
+            var request = new HttpRequestMessage(HttpMethod.Get, "http://localhost/OnGetView?message=functionaltests");
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs]: What's `message` for?

File: test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs [hunk 2]
```diff
@@ -112,6 +112,22 @@ public RazorPagesTest(MvcTestFixture<RazorPagesWebSite.Startup> fixture)
         }
 
         [Fact]
+        public async Task ViewReturnsPage()
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs]: Could we get another test with a `void OnGet`? Unless that'a already covered.

File: test/WebSites/RazorPagesWebSite/OnGetView.cshtml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+@functions {
+    public IActionResult OnGet()
+    {
+        Message = "From OnGet";
+
+        return View();
+    }
+    public string Message { get; set; } = "Default";
+}
+The message: @Message
```
Comment (pranavkm) [inline test/WebSites/RazorPagesWebSite/OnGetView.cshtml]: New line at the end of file.

File: test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs (hunk 2 above)
Comment (ryanbrandenburg) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs]: [This](https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs#L69) test covers `void OnGet` by hitting [here](https://github.com/aspnet/Mvc/blob/dev/test/WebSites/RazorPagesWebSite/HelloWorldWithHandler.cshtml#L6).

File: test/WebSites/RazorPagesWebSite/OnGetView.cshtml (hunk 3 above)
Comment (ryanbrandenburg) [inline test/WebSites/RazorPagesWebSite/OnGetView.cshtml]: Added

**Comment by ryanbrandenburg** [comment]
:up::date:

**Comment by ryanbrandenburg** [comment]
@pranavkm is there a :shipit: from you?

**Pull request merged**